components/desktop/mate/libmatemixer/pkg5

```diff
@@ -3,6 +3,7 @@
         "SUNWcs",
         "library/audio/pulseaudio",
         "library/glib2",
+        "shell/ksh93",
         "system/header/header-audio",
```

Comment:
If you add new REQUIRED_PACKAGES in the Makefile corresponding entries in pkg5 are expected.

Reply:
As soon as you have fixed pkg5 I will start integrating the mate PR's. I don't think that we will find more testers here and it runs stable enough for me on my desktop.

Reply:
Over today I cherry-pick the patches and build them one by one and eventually upgraded my server(s) and from the first basic testing the thing I use daily work. The one running my SunRays was the most important and there I have not seen anything that works to a degree worse then the previous version. So for me also a +1 as to integrating this.

Reply:
Good to know that we have someone who is really interested in Mate tests ;-)

Reply:
Thanks @mvwieringen for the feedback, it's very useful.

I am concerned that there may be a memory corruption or uninitialized value somewhere in `caja` that's causing the one issue that @AndWac noted, but until I can reproduce it, tracking down the problem is difficult.  If it's very rare and no one else triggers it, I guess that's OK, but if not I'm hoping someone can provider a reproducer.